README.md

```diff
@@ -1,18 +1,61 @@
-# Home Assistant Operating system
+# Home Assistant Operating System
 
-Home Assistant Operating System (HassOS) is based on [buildroot](https://buildroot.org/). It's a hypervisor for Docker and supports various kind of hardware. It is also available as virtual appliance for different virtualization solutions. The whole system is optimized for hosting [Home Assistant](https://www.home-assistant.io) and its features (to be precise, the [Add-ons](https://www.home-assistant.io/addons/)). You can update the system by using OTA updates or offline updates.
+Home Assistant Operating System (HassOS) is an operating system optimized for hosting [Home Assistant](https://www.home-assistant.io) and its [Add-ons](https://www.home-assistant.io/addons/).
 
-This is an embedded Linux which works different than a normal Linux distribution. The system is designed to run with minimal I/O and is optimized for its tasks.
+HassOS is uses Linux as a hypervisor for Docker, allowing each component of Home Assistant including add-ons to run in separate docker containers. It uses embedded Linux based on [buildroot](https://buildroot.org/), which is different to a "normal" Linux, and it is optimized for running Home Assistant on IoT devices.
```

Comment:
> ...and it is optimized for running Home Assistant on IoT devices.

Hmmm, Home Assistant doesn't run on IoT devices per se. SBC can be IoT devices but you can't run Home Assistant on a temperature sensor.

> which is different to a "normal" Linux

It's different to a classical Linux distribution. Still Linux.

Reply:
Agreed. I’d left these words in since they were from the original author of the readme. I’ll change them based on your suggestions.

Reply:
Resolved in commit e1cc5f461d35c1881c41122352d11fe0d184d07e